src/types/module.rs

```diff
@@ -194,11 +195,50 @@ impl PyModule {
     /// ```rust,ignore
     /// m.add("also_double", wrap_pyfunction!(double)(py));
```

Comment:
arguments are `(py)` here, but `(py, m)` below?

Reply:
I missed these because the tests are ignored, I'll go through the guide again to make the `add_X` calls consistent. The correct way to add a function is to pass the `PyModule` to the wrapper. `Python` can be acquired from  `PyModule`.

Reply:
`(py, m)` was an outdated design we removed. `ignore` docstring example probably was a mistake.

`wrap_pyfunction!` output can take either `py` (for backwards compatibility, will deprecate eventually) or `m` (preferred, sets the `__module__` correctly).

Reply:
OK, thanks!